services/brig/test/integration/API/User/Account.hs

```diff
@@ -157,8 +154,6 @@ tests _ at opts p b c ch g aws userJournalWatcher =
       test p "delete/by-code" $ testDeleteUserByCode b,
       test p "delete/anonymous" $ testDeleteAnonUser b,
       test p "delete with profile pic" $ testDeleteWithProfilePic b ch,
-      test p "delete with connected remote users" $ testDeleteWithRemotes opts b,
-      test p "delete with connected remote users and failed remote notifcations" $ testDeleteWithRemotesAndFailedNotifications opts b c,
```

Comment:
The first test is already being tested in end-to-end tests. It is very hard to test in the integration tests.

The second case is not valid anymore because we cannot fail due to remote backend being down, we can fail due to rabbitmq being down, but that is just a 500. I didn't think we should test 500 scenarios.